packages/contracts-bedrock/src/Safe/VetoerSet/PrivilegedAddOwnerModule.sol

```diff
@@ -0,0 +1,60 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.15;
+
+import { Safe } from "safe-contracts/Safe.sol";
+import { Enum } from "safe-contracts/common/Enum.sol";
+
+import { ISemver } from "src/universal/ISemver.sol";
+
+import { OwnerGuard } from "./OwnerGuard.sol";
+
+/// @title PrivilegedAddOwnerModule
+/// @notice This module allows any specifically designated address to add owners to the
+///         Safe Account. Specifically, the Optimism Foundation may add new owners.
+contract PrivilegedAddOwnerModule is ISemver {
```

Comment:
Are you open to just `AddOwnerModule`? `Privileged` feels excess in this case since the module is small enough and self-explanatory enough